tests/integration/standard/column_encryption/test_policies.py

```diff
@@ -34,7 +36,7 @@ def _recreate_keyspace(self, session):
 
     def _create_policy(self, key, iv = None):
         cl_policy = AES256ColumnEncryptionPolicy()
-        col_desc = ColDesc('foo','bar','encrypted')
+        col_desc = ColDesc('foo', ' bar', 'encrypted')
```

Comment:
@bschoening Went back and looked again and this change is the reason the CLE tests are failing on your branch.  The addition of the extra space here means there's no a mismatch between the actual column name and what's specified in the CLE policy which causes the encryption to behave weirdly.  Undoing just this one change caused the test to pass for me locally.